### ISSUE TITLE
Ask for review from people being added/removed in META.yml

### ISSUE THREAD

**Issue by zcorpan** [web-platform-tests/wpt-pr-bot#32]
and omit the "There are no reviewers for this pull request." comment.

**Comment by tobie** [comment]
To answer your security related question, we can’t have folks who aren’t yet reviewers review the PR that would make them a reviewer. That’s too easy to game.

**Comment by zcorpan** [comment]
OK, that makes sense. I agree we shouldn't make people reviewers automatically, only request review. The bot could ask one of the admins to update the reviewers team if necessary?

**Comment by zcorpan** [comment]
Yes, good point. The bot can detect that and mention them plus an admin
